tasks/get-ocp-version/tests/mocks.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eux
+
+function oras() {
```

Comment:
Resolving but reminder @theflockers to please address this in a followup PR. I don't want it done in this one because then this code won't have gone through e2e

Reply:
this one is already removed.

Reply:
It wasn't when this merged, hence the comment